examples/widget-gallery/src/clipboard.rs

```diff
@@ -21,7 +21,9 @@ pub fn clipboard_view() -> impl View {
             }),
             form_item("Copy from input".to_string(), 120.0, move || {
                 h_stack((
-                    text_input(text1).keyboard_navigatable(),
+                    text_input(move || text1.get()).on_update(move |val| {
```

Comment:
Should presumably still be `.keyboard_navigatable`

Reply:
I've moved it  to the widget fn(since an input without `keyboard_navigatable` is not really functional), so user code does not have to specify it.